.github/workflows/ci.yml

```diff
@@ -225,3 +225,13 @@ jobs:
         path: |
           code-coverage-results.md
           pull_request_id
+
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      # https://github.com/marketplace/actions/pr-labeler-based-on-multiple-rules
+      - uses: srvaroa/labeler@v0.9
```

Comment:
Why not use the [GitHub-provided labeler action](https://github.com/actions/labeler)?

Reply:
Ah, I see, the action allows a richer set of conditions. Please update the above comment to indicate why the non-GH labeler action is used. Also, please change the link to https://github.com/srvaroa/labeler as it is shorter and takes us directly to the action's source in case we ever need to look into it.

Reply:
Added comments in the latest patch.